### PR TITLE
updating CHANGELOG.md to reflect 5.0 changes

### PR DESCRIPTION
**Pull request by cstolli**
#PATCH#
updating CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,106 @@
+# 5.0
+## Breaking Changes
+* **Removed** support for `PascalCaseCustomRenderers` in view schema definitions
+* **Added** support for `kebab-case-custom-renderers` in view schema definitions
+  this was done mainly to accommodate a terser DSL in view schemas for built-in
+  renderers such as `select` which was added in this version. The goal is to have
+  simple built-in renderers be targeted with easy names like `select`, `buckets`, etc
+  like they were in UI Schema. UI Schema 2 will follow suit as well.  True custom renderers
```
Comment (sandersky) [inline CHANGELOG.md]: Referencing UI Schema on an open source project will likely confuse people. This description should probably be revised a little bit.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,106 @@
...
+  (created in consuming code) can still be addressed either way (based on how the
+  consuming code constructs the custom renderers hash), but the kebab-case is encouraged.
+
+## Non-Breaking Changes
+* **Added** support for built-in select drop-downs, using `ember-frost-select`.
```
Comment (sandersky) [inline CHANGELOG.md]: `ember-frost-select` isn't actually a thing anymore. Instead I'd say using `frost-select` from `ember-frost-core`.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,106 @@
...
+  ```JSON
+  ...
+  {
+    "model": "someProperty",
+    "renderer": "select"
+  }
+  ...
+  ```
+
+  Where options are a based on an API call, arguments to an `Ember.store.query` call
```
Comment (sandersky) [inline CHANGELOG.md]: Remove the word `a` after `are`.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,106 @@
+# 5.0
+## Breaking Changes
+* **Removed** support for `PascalCaseCustomRenderers` in view schema definitions
+* **Added** support for `kebab-case-custom-renderers` in view schema definitions
+  this was done mainly to accommodate a terser DSL in view schemas for built-in
```
Comment (gknoy) [inline CHANGELOG.md]:
```diff
-  this
+  This
```

**Comment by sandersky** [comment]
:+1:

**Comment by sandersky** [comment]
:+1:

**Pull request merged**